src/terra-dev-site/example/ResponsiveListAndCellGridExample.jsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import { ActiveBreakpointContext } from 'terra-application/lib/breakpoints';
+import IconPerson from 'terra-icon/lib/icon/IconPerson';
+import IconAlert from 'terra-icon/lib/icon/IconAlert';
+import IconInformation from 'terra-icon/lib/icon/IconInformation';
+import IconAttachment from 'terra-icon/lib/icon/IconAttachment';
+import List, { Item } from 'terra-list';
+import ItemView from 'terra-clinical-item-view';
+import CellGrid, { Cell } from 'terra-cell-grid';
+import classNames from 'classnames/bind';
+import styles from './ResponsiveListAndCellGridExample.scss';
+import mockData from './mock-data/cell-grid-data';
+
+const display1 = <ItemView.Display icon={<IconPerson />} text="Asif Khan" />;
+const display2 = <ItemView.Display text="Care Position: Primary" />;
+const display3 = <ItemView.Display text="Room 100A" />;
+const display4 = <ItemView.Display text="Acuity: 5" />;
+const display5 = <ItemView.Display text="Start Time: 08-05-2016 12:00:00" />;
+const display6 = <ItemView.Display text="End Time: 08-05-2016 16:00:00" />;
+const comment = <ItemView.Comment text="Faint red rash appeared at 08-05-2016 13:24:00" />;
+const startAccessory = <IconAlert />;
+const endAccessory = <IconInformation />;
+
+const cx = classNames.bind(styles);
+
+const listDisplay = (
+  <List role="listbox">
```

Comment:
Clean up the list implementation like you did the table and use the same mock data for both.

Reply:
Fixed https://github.com/cerner/terra-ui/pull/310/commits/e9da608d29aaca7a6a54bf05797065b88e72b842#diff-db8ae280acc81345eeba3a440c2c81285c34457faf9cbfb01de5508791e3283fR16